features/step_definitions/ab_testing_steps.rb

```diff
@@ -35,8 +37,18 @@
 end
 
 Then(/^the bucket is reported to Google Analytics$/) do
-  # TODO: Implement once we know how we're reporting to GA. Is it sufficient to
-  # check the meta tag which is inspected by GA to report the page view?
+  analytics = []
+
+  Timeout.timeout(Capybara.default_max_wait_time) do
+    analytics = page_track_requests until analytics.length > 0
+  end
+
+  assert analytics.length == 1, "Expected exactly 1 page track request"
+
+  query = Rack::Utils.parse_query URI(analytics.first.url).query
+  expected_query = "Example:#{@ab_cookie_value}"
+  assert query['cd40'] == "Example:#{@ab_cookie_value}",
+    "Expected a query string cd40=#{expected_query} but instead found cd40=#{query['cd40']}"
```

Comment:
You can get rid of most of this message if you use the RSpec expectation `foo.should == bar` or the MiniTest assertion `assert_equal(foo, bar, "some message"`), which give more informative default failure messages.